native-image/native-image-quick-start/README.md

```diff
@@ -0,0 +1,363 @@
+<img src="https://www.graalvm.org/resources/img/home/logo_mobile_openmenu.svg"
+alt="GraalVM logo"
+width="200px">
+
+# Get Started with GraalVM Native Image
```

Comment:
@krisfoster, I upgraded and refactored this workshop to be compatible with GraalVM for JDK 24, latest Native Build Tools and the demo part has to be replaced.